plugins/catalog-import/package.json

```diff
@@ -40,6 +40,7 @@
     "@backstage/errors": "workspace:^",
     "@backstage/integration": "workspace:^",
     "@backstage/integration-react": "workspace:^",
+    "@backstage/plugin-catalog-backend": "workspace:^",
```

Comment:
freben is right that we can't depend on the backend package in the frontend. There's two options, either move the shared type(s) out to `@backstage/plugin-catalog-common`. The package need to be created too and in the future we likely want to place more types here so it might be a good thing for the future. Alternatively duplicate the type in the frontend package if that's a small change and slap a TODO on it.